R/factor_analysis.R

```diff
@@ -19,27 +20,40 @@
 #'   efa <- factor_analysis(mtcars[, 1:5], n = 2)
 #'   summary(efa)
 #'   predict(efa)
-#' \donttest{
```

Comment:
Why did the tool change the examples here? This doesn't increase credibility of that tool. ;-)

Reply:
Oh, this is not something `styler` did. I did that because `styler` currently can't handle `donttest` nested inside `if` loop. So I just added everything to `donttest`. 

This is not so crucial, right? Even if the example is not tested, the tests are cover this function anyway.

Reply:
I'm not sure `summary()` and `predict()` are included in tests, so we should either add a test or don't wrap those example parts in `\donttest{}`